### PR TITLE
Add marketing processes to the Handbook.md

### PR DESCRIPTION
**Pull request by robmarcer**
First pass at objectively defining our comms channels and content in the handbook.

### CODE REVIEW

**Comment by robmarcer** [comment]
I wonder if this should move out of the design section but I added if here for now.

**Comment by ZJvandeWeg** [comment]
> I wonder if this should move out of the design section but I added if here for now.

Think we might need a section indeed, like `marketing` in the top level?

File: design/index.md [hunk 1]
```diff
@@ -42,9 +42,95 @@ Other tools included in the Creative Cloud suite that may be useful include:
...
+## Websites
 
 - All written content should be in UK English.
 - All page titles should summarise the content, keep the URL length as short as practical and use [Kebab Case](https://en.wiktionary.org/wiki/kebab_case).
 - All images should use informative [alt tags](https://www.w3.org/WAI/tutorials/images/tips/) which clearly describe the point of an image rather than all the details. Alt tags should be no longer than 60 characters.
 - When mentioning [FlowForge Concepts](https://flowforge.com/docs/user/concepts/) (terminology) where possible we should link to an explanation of that concept.
+
+## Social Media, Marketing & PR
+
+IT is important we understand what we communicate, to who, how and when. This sets out our principles for social media, marketing and PR communications.
```
Comment (ZJvandeWeg) [inline design/index.md:54]:
```suggestion
It's important we understand what we communicate, to who, how, and when. This sets out our principles for social media, marketing, and PR communications.
```

File: design/index.md [hunk 2]
```diff
@@ -42,9 +42,95 @@ Other tools included in the Creative Cloud suite that may be useful include:
...
+
+As soon as we are aware of planned downtime we should publish the information, that should be at least one week before the downtime.
+
+#### Outage
+
+If we experience unplanned downtime we should communicate in near real time to keep our users and stakeholders up to date with the situation.
+
+#### Corporate Comms
+
+Major company news such as investment rounds should be communicated at the time agreed with the relevant parties involved, it is important there is no significant delay in communicating this information.
```
Comment (ZJvandeWeg) [inline design/index.md:86]:
```suggestion
Major company news such as investment rounds should be communicated at the time agreed with the relevant parties involved. We strive to make this information public as soon as practical and effective.
```

This is ambiguous as a sentence, does our audience not experience a delay, or inter party communication?

File: design/index.md [hunk 3]
```diff
@@ -42,9 +42,95 @@ Other tools included in the Creative Cloud suite that may be useful include:
...
+
+### Content Channels
+
+Each content type has channels which are appropriate, this table sets out which content should be posted on each channel.
+
+|Content          |Blog  |Mailing List |Twitter  |Youtube  |LinkedIn |Reddit |Node-RED Slack|
+|---              |---   |---          |---      |---      |---      |---    |---           |
+|Release          |*     |             |*        |*        |*        |*      |*             |  
+|Patch            |      |             |*        |         |         |*      |*             |
```
Comment (ZJvandeWeg) [inline design/index.md:95]: I think a patch release should require a blog post. Unlike the release blog post, the tone is more factual and much shorter. But it does need publishing what was fixed and improved. These changes might require specific update instructions too.

File: design/index.md [hunk 4]
```diff
@@ -42,9 +42,95 @@ Other tools included in the Creative Cloud suite that may be useful include:
...
+
+Each content type has channels which are appropriate, this table sets out which content should be posted on each channel.
+
+|Content          |Blog  |Mailing List |Twitter  |Youtube  |LinkedIn |Reddit |Node-RED Slack|
+|---              |---   |---          |---      |---      |---      |---    |---           |
+|Release          |*     |             |*        |*        |*        |*      |*             |  
+|Patch            |      |             |*        |         |         |*      |*             |   
+|Newsletter       |*     |*            |*        |         |*        |*      |*             |
+|How To           |      |*            |*        |         |*        | *     |*             |  
+|Planned Downtime |      |*            |*        |         |*        |*      |*             |
```
Comment (ZJvandeWeg) [inline design/index.md:98]:
```suggestion
|Planned Downtime | *     |?            |*        |         |*        |*      |*             |  
```

I think our mailing list is the highlights of the past 30 days or so. Planned downtime might occur in before a newsletter goes out too?

Or is the Mailing list not the same as the monthly newsletter to the mailing list?

File: design/index.md [hunk 5]
```diff
@@ -42,9 +42,95 @@ Other tools included in the Creative Cloud suite that may be useful include:
...
+
+Content should be eloquent and an appropriate length to communicate the message.
+
+#### Mailing List
+
+Content should be eloquent and an appropriate length to communicate the message.
+
+#### Twitter
+
+Keep it short, Tweets should be factual, informal and have a call to action (a link) where a user can read more.
```
Comment (ZJvandeWeg) [inline design/index.md:114]: I'd also say this is an opportunity to be more fun and not too corporate in tone. A blog is more formal for example.

File: design/index.md [hunk 6]
```diff
@@ -42,9 +42,95 @@ Other tools included in the Creative Cloud suite that may be useful include:
...
+
+- Include opening and closing idents
+- Be clear in what they are communicating.
+- Have an appropriate thumbnail.
+- Have a call to action in the description.
+- Be publicly listed.
+
+#### LinkedIn
+
+Posts should be framed in the context of FlowForge as an organisation and a team, don't get too technical. A relevant photo or image should be included in every post. Where the post is primarily of a video it should be embedded from Youtube.
```
Comment (ZJvandeWeg) [inline design/index.md:128]: 👌

**Review by ZJvandeWeg — CHANGES_REQUESTED** [review]

**Comment by ZJvandeWeg** [comment]
@robmarcer Can you update the title of the PR?

**Comment by joepavitt** [comment]
> Think we might need a section indeed, like marketing in the top level? 

Agreed, I am in the process of adding another top-level item around video content anyway, and so, with this in, I can work to re-organise sensibly.

File: design/index.md (hunk 4 above)
Comment (robmarcer) [reply design/index.md:98]: Sorry, this was an error in my MD table, I don't think we should send emails about planned downtime.

File: design/index.md [hunk 7]
```diff
@@ -42,9 +42,95 @@ Other tools included in the Creative Cloud suite that may be useful include:
...
 - All images should use informative [alt tags](https://www.w3.org/WAI/tutorials/images/tips/) which clearly describe the point of an image rather than all the details. Alt tags should be no longer than 60 characters.
 - When mentioning [FlowForge Concepts](https://flowforge.com/docs/user/concepts/) (terminology) where possible we should link to an explanation of that concept.
+
+## Social Media, Marketing & PR
+
+It's important we understand what we communicate, to who, how, and when. This sets out our principles for social media, marketing, and PR communications.
+
+### Content Types
+
+Content should be published to the appropriate channels, the content we create is as follows.
```
Comment (joepavitt) [inline design/index.md:58]:
```suggestion
Content should be published to the appropriate channels, the types of published content we currently produce are as follows.
```

File: design/index.md [hunk 8]
```diff
@@ -42,9 +42,95 @@ Other tools included in the Creative Cloud suite that may be useful include:
...
+#### Youtube
+
+Videos should:
+
+- Include opening and closing idents
+- Be clear in what they are communicating.
+- Have an appropriate thumbnail.
+- Have a call to action in the description.
+- Be publicly listed.
+
```
Comment (joepavitt) [inline design/index.md:125]: Further details will be found in the (to be reviewed/merged) `/handbook/media` section

File: design/index.md (hunk 4 above)
Comment (robmarcer) [reply design/index.md]: Hopefully resolved in https://github.com/flowforge/handbook/pull/133/commits/a01d7df2db115203663ec7ed363ee0049aecdb30

File: design/index.md (hunk 3 above)
Comment (robmarcer) [reply design/index.md:95]: I've made this change.

File: design/index.md [hunk 9]
```diff
@@ -42,9 +42,95 @@ Other tools included in the Creative Cloud suite that may be useful include:
...
+#### LinkedIn
+
+Posts should be framed in the context of FlowForge as an organisation and a team, don't get too technical. A relevant photo or image should be included in every post. Where the post is primarily of a video it should be embedded from Youtube.
+
+#### Reddit
+
+Keep it short, Reddit posts should be factual, informal and have a call to action (a link) where a user can read more.
```
[the remark this answers is not in the document]
Comment (robmarcer) [reply design/index.md:132]: I've made this change.

File: design/index.md [hunk 10]
```diff
@@ -42,9 +42,95 @@ Other tools included in the Creative Cloud suite that may be useful include:
...
+
+#### Newsletter
+
+Monthly newsletter published at the start of each calendar month covering news from the prior month.
+
+#### How To
+
+Technical guides, published once per week.
+
+#### Planned Downtime
```
[the remark this answers is not in the document]
Comment (robmarcer) [reply design/index.md:76]: I've updated the order as per this suggestion.

**Review by knolleary — APPROVED** [review]

**Review by joepavitt — APPROVED** [review]

**Pull request merged**